app/vep/model.py

```diff
@@ -0,0 +1,81 @@
+# generated by datamodel-codegen:
+#   filename:  APISpecification.yaml
+#   timestamp: 2024-06-10T15:52:34+00:00
+
+from __future__ import annotations
+
+from enum import Enum
+from typing import Any, List, Optional, Union
+
+from pydantic import BaseModel, Field
+
+class PaginationMetadata(BaseModel):
+    page: int
+    per_page: int
+    total: int
+
+class PredictedIntergenicConsequence(BaseModel):
+    feature_type: Any = Field(
```

Comment:
This field's value is always a null. The model is being too permissive here.

Reply:
Currently providing Null in the code. Will look at the model generator to see how to support this